pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java

```diff
@@ -229,18 +231,29 @@ private String buildModule(Module base, ModuleVersion moduleVersion, Map<String,
             asciidoctor.javaExtensionRegistry().includeProcessor(
                     new SlingResourceIncludeProcessor(base));
             asciidoctor.javaExtensionRegistry().postprocessor(
-                    new HtmlModulePostprocessor(base));
+                    new HtmlModulePostprocessor(base));            
 
             // add specific extensions for metadata regeneration
             if(regenMetadata) {
                 asciidoctor.javaExtensionRegistry().treeprocessor(
                         new MetadataExtractorTreeProcessor(moduleVersion.metadata().getOrCreate()));
-            }
+            }                       
 
             String html = "";
-            try {
+            try {                
+                String ascContent = moduleVersion.content().get().asciidocContent().get();                
+                
+                //Extracting link from the adoc using preprocessor        
+                asciidoctor.javaExtensionRegistry().preprocessor(
+                        new UuidPreProcessor(base));
+                html = asciidoctor.convert(
+                        ascContent,
+                        OptionsBuilder.options().toFile(false));
+                
+                log.info("asciidoctor content: {}", html);
```

Comment:
Please remove these lines. You are calling `.convert()` twice here - the first result gets overwritten by the second, so it must be unnecessary. We probably don't need the `log` statement here either.